### DISCUSSION TITLE
Git author must have access to the project on Vercel

### DISCUSSION THREAD

**Discussion by williamaditama** [vercel/community#1126] [Help] [answered]
When creating a PR, I am getting this error, preventing me from merging:
![image](https://user-images.githubusercontent.com/23274070/206414561-80a50a2b-d1df-4fc8-a2e1-bd0977493880.png)

**Comment by amyegan** [comment]
Sorry you're running into this problem @williamaditama. Can you confirm that you are already a member of the Team that owns the project, and you are using the same email on your Vercel and GitHub accounts?

If all of that is set and you're still facing this issue, please share a deployment ID, commit SHA and/or your account slug so we can look into the logs to get a better idea of what is going wrong 🙏

**Reply by dawoodEsketchers97** [reply]
hi @amyegan ,
I am facing this problem. I am member of the Team that owns the project and using the same email on Vercel and Github.

The commit SHA is fc3d7352f6a77073b082d9b70b789ea61f520966. There was no deployment so cant share deployment ID.
Your help is appreciated.

Regards,
Dawood

![image](https://user-images.githubusercontent.com/94681922/214086896-8682f113-8c6d-4220-a18c-d92a6557235f.png)

**Reply by amyegan** [reply, marked as answer]
Sorry you're running into trouble with this @dawoodEsketchers97. It looks like you already checked that you have the same email address on GitHub and Vercel. The next steps are to check that your local environment has [Correctly Configured Metadata](https://vercel.com/guides/why-aren-t-commits-triggering-deployments-on-vercel#correctly-configured-metadata) and to disconnect and re-connect your GitHub account in your [Login Connections](https://vercel.com/account/login-connections). Please let me know if you continue to see this problem

**Reply by dawoodEsketchers97** [reply]
Thanks for the support. The error was due to some error during the build process.

**Comment by rdylina** [comment]
@amyegan I'm having the same problem.

no deployment ID because the branch won't deploy. Here is the commit SHA 2c22f892d84428dd2a87a05c39433f579ec1acd5

**Reply by rdylina** [reply]
@amyegan update. it was the git metadata for username and email. They didn't match github. Would be great to have a check specifically for this and if it recommended a solution as part of the error tooltip. This was pretty obscure and hard to track down since you can't view a commit's metadata anywhere on github that I am aware of.

**Comment by yakcmxela** [comment]
@amyegan 

I am no longer able to create deployments via commits to our BitBucket repo. Nothing has changed with my account and it was working yesterday. I've tried disconnecting / reconnecting both the project and my own account to BitBucket but no luck. My git config email and name match my BitBucket account information. My account has been removed and re-added to the team by the team owner. Not sure what else to try. Any ideas?

Project slug: gowild-web
Commit: cc02d0632c007bd372010f44318f407114accd30

**Reply by S3gillu** [reply]
@yakcmxela We are also facing the same issue, the deployment trigger with Bitbucket suddenly stopped working.

**Reply by susilthapa** [reply]
Facing same issue

**Reply by juicyjusung** [reply]
Facing same issue

**Reply by alahji7** [reply]
Same here...

**Reply by juicyjusung** [reply]
I think there's an issue with authentication in the vercel and bitbucket integration, I had no issues with github. T.T

**Reply by Andy-Reynolds** [reply]
Same issue

**Reply by fmaschewski** [reply]
Same here

**Reply by miller-design** [reply]
same issue here as well

**Reply by idChef** [reply]
Same here

**Reply by hjbrand** [reply]
Same here

**Reply by Mr-lsaidi** [reply]
same here

**Reply by mrtnbroder** [reply]
same here

**Reply by swarnava** [reply]
Hi,

We have pushed a fix for this issue. You can redeploy and give it a try. :)

**Reply by 0xDiamondCutter** [reply]
Having similar issue. Today my commits just stopped being deployed on Vercel and the builds failed due to "Git author must have access to the project on Vercel." I'm the owner of the repo, logged in with bitbucket and I changed nothing in settings.

**Reply by amyegan** [reply]
Thanks for reporting this. A fix was shipped minutes ago and automatic deployments should start working normally again on your next new commit. Please let me know if you continue to see this problem.

**Reply by yakcmxela** [reply]
Working now, thank you!

**Reply by Bayyn** [reply]
Hello @amyegan ,
i am having the same problem. There is no problem in the projects in my staff account, but the projects in my team account do not receive the last commits.
